LDAR_Sim/model_code/OGI_crew.py

```diff
@@ -135,9 +127,9 @@ def work_a_day(self):
 
         if self.worked_today:
             self.timeseries['OGI_cost'][self.state['t'].current_timestep] += \
-                self.config['cost_per_day']
+                self.config['cost']['per_day']
             self.timeseries['total_daily_cost'][self.state['t'].current_timestep] += \
-                self.config['cost_per_day']
+                self.config['cost']['per_day']
```

Comment:
Did you make it so that every option can actually accrue costs 'up front', 'per site', 'per day', etc? If not, isn't it a bit dangerous to have functionality that doesn't do anything without warnings? Ideally, this commit would enable all solutions to charge a cost per day or a cost per site...